### PR TITLE
Ledger: Remove non-working websocket 2FA for legacy ledger devices

### PR DESCRIPTION
**Pull request by EchterAgo**
The service is not available anymore and the app has been removed from the Play store. This fixes the 2FA dialog for the Windows build as the websocket module is not included.

### CODE REVIEW

**Comment by EchterAgo** [comment]
Sorry, don't merge this yet. I forgot to test this with the built installer.

**Comment by cculianu** [comment]
Awesome man. Just let me know what it's ready.

File: plugins/ledger/auth2fa.py [hunk 1]
```diff
@@ -149,205 +124,32 @@ def pin_changed(s):
...
-            self.accept()
-
     def getDevice2FAMode(self):
         apdu = [0xe0, 0x24, 0x01, 0x00, 0x00, 0x01] # get 2fa mode
         try:
             mode = self.dongle.exchange( bytearray(apdu) )
             return mode
         except BTChipException as e:
-            debug_msg('Device getMode Failed')
+            print_msg('Device getMode Failed')
```
Comment (cculianu) [inline plugins/ledger/auth2fa.py]: Where is this symbol coming from?  I don't see it from imports.

`electroncash.util` has a `print_msg` but that is printing to stdout unconditionally.  Usually we do `print_error` or we inherit from `util.PrintError` and do `self.print_error`...

Comment (cculianu) [reply plugins/ledger/auth2fa.py]: Entirely your call. ha. :)

Comment (EchterAgo) [reply plugins/ledger/auth2fa.py]: Yea, I'll switch it to `print_error`. Also looks like I still need to import `BTChipException`.

**Comment by EchterAgo** [comment]
Updated

**Comment by EchterAgo** [comment]
Yup, 2FA on Windows in the pyinstaller build works now.

**Comment by cculianu** [comment]
Awesome. Merging.

**Pull request merged**